client/plots/singleCellPlot.js

```diff
@@ -774,8 +774,8 @@ class singleCellPlot {
 		plot.colorGenerator = d3Linear().domain([min, max]).range(colors)
 
 		const gradientWidth = 100
-		const gradientStep = (max - min) / 4
-		const tickValues = [min < 0 ? min : 0, min + gradientStep, min + 2 * gradientStep, min + 3 * gradientStep, max]
+		const gradientStep = (max - min) / 3
+		const tickValues = [min < 0 ? min : 0, min + gradientStep, min + 2 * gradientStep, max]
```

Comment:
incorrect logic and ends up with wrong axis. please do not generate tick values but let d3-axis do it automatically
<img width="160" alt="image" src="https://github.com/user-attachments/assets/0e5cb1b7-df78-4ebd-8a4e-f68ceba25728">